selenium/che-selenium-test/src/test/resources/conf/selenium.properties

```diff
@@ -24,3 +24,17 @@ github.auxiliary.password=
 # Google credentials
 google.user=
 google.password=
+
+# Define configuration for che default test user.
+# Note that for single user assembly the configuration should be:
```

Comment:
It is n't clear if use has to be predefined in single user assembly. Could you clarify pls

Reply:
added default values, so for single user che it's not needed to provide default user data